frontend/src/pages/NewPipelineVersion.test.tsx

```diff
@@ -42,39 +41,17 @@ describe('NewPipelineVersion', () => {
   let getPipelineSpy: jest.SpyInstance<{}>;
   let createPipelineSpy: jest.SpyInstance<{}>;
   let createPipelineVersionSpy: jest.SpyInstance<{}>;
-  let uploadPipelineSpy: jest.SpyInstance<{}>;
+  let uploadPipelineVersionSpy: jest.SpyInstance<{}>;
 
   let MOCK_PIPELINE = {
-    id: 'original-run-pipeline-id',
-    name: 'original mock pipeline name',
-    default_version: {
-      id: 'original-run-pipeline-version-id',
-      name: 'original mock pipeline version name',
-      resource_references: [
-        {
-          key: {
-            id: 'original-run-pipeline-id',
-            type: ApiResourceType.PIPELINE,
-          },
-          relationship: 1,
-        },
-      ],
-    },
+    pipeline_id: 'original-run-pipeline-id',
+    display_name: 'original mock pipeline name',
   };
 
   let MOCK_PIPELINE_VERSION = {
-    id: 'original-run-pipeline-version-id',
-    name: 'original mock pipeline version name',
+    pipeline_version_id: 'original-run-pipeline-version-id',
+    display_name: 'original mock pipeline version name',
     description: 'original mock pipeline version description',
-    resource_references: [
```

Comment:
While removing resource-reference, you need to keep the pipeline ID info under `pipeline_id`, so the mock should look like:
```typescript
let MOCK_PIPELINE_VERSION = {
    pipeline_version_id: 'original-run-pipeline-version-id',
    display_name: 'original mock pipeline version name',
    description: 'original mock pipeline version description',
    pipeline_id: 'original-run-pipeline-id',
}
```